microprofile-fault-tolerance/src/test/java/org/jboss/eap/qe/microprofile/fault/tolerance/v20/FaultTolerance20AsyncTest.java

```diff
@@ -89,7 +87,8 @@ public void timeoutFailureCompletionStage() {
     }
 
     /**
-     * @tpTestDetails Deploy MP FT application with 1 second @Timeout, @BulkHead (hystrix.threadpool.default.maximumSize=40)
+     * @tpTestDetails Deploy MP FT application with 1 second @Timeout, @BulkHead
+     *                (io.smallrye.faulttolerance.globalThreadPoolSize=40)
```

Comment:
There is a difference against Hystrix. `io.smallrye.faulttolerance.globalThreadPoolSize` does not include bulkhead thread pools. The test needs re-work. Maybe it will be unnecessary after https://github.com/smallrye/smallrye-fault-tolerance/issues/198 fix (if there will be corresponding TCK test).

Reply:
I updated the tests. I removed usage of io.smallrye.faulttolerance.globalThreadPoolSize since it wasn't necessary in any of the tests.